test/regress/run_regression.py

```diff
@@ -79,6 +80,29 @@ def get_cvc4_features(cvc4_binary):
     return features
 
 
+def logic_supported_with_proofs(logic):
+    return logic in [
```

Comment:
Maybe add an `assert isinstance(logic, str)` since we want to make sure that we are not returning `False` just because `logic` has the wrong type.

Reply:
Added.